internal/policies/mount/mount_test.go

```diff
@@ -67,25 +67,25 @@ func TestApplyPolicy(t *testing.T) {
 	}{
 		/***************************** USER ****************************/
 		// Success cases.
-		"successfully apply policy for entry with one value":        {},
-		"successfully apply policy for entry with multiple values":  {entries: []string{"entry with multiple values"}},
-		"successfully apply policy for entry with repeatead values": {entries: []string{"entry with repeatead values"}},
-		"successfully apply policy filtering out unsupported keys":  {entries: []string{"entry with multiple values", "entry with one value"}, keys: []string{"unsupported", "user-mounts"}},
+		"user, successfully apply policy for entry with one value":        {},
+		"user, successfully apply policy for entry with multiple values":  {entries: []string{"entry with multiple values"}},
+		"user, successfully apply policy for entry with repeatead values": {entries: []string{"entry with repeatead values"}},
+		"user, successfully apply policy filtering out unsupported keys":  {entries: []string{"entry with multiple values", "entry with one value"}, keys: []string{"unsupported", "user-mounts"}},
 
 		// Special cases.
-		"successfully apply policy with anonymous values":     {entries: []string{"entry with anonymous tags"}},
-		"creates only users_user dir if the entry is errored": {entries: []string{"errored entry"}},
+		"user, successfully apply policy with anonymous values":     {entries: []string{"entry with anonymous tags"}},
+		"user, creates only users_user dir if the entry is errored": {entries: []string{"errored entry"}},
 
 		// Badly formatted entries.
-		"successfully apply policy trimming whitespaces":           {entries: []string{"entry with spaces"}},
-		"successfully apply policy trimming sequential linebreaks": {entries: []string{"entry with multiple linebreaks"}},
-		"creates only users_user dir if the entry is empty":        {entries: []string{"entry with no value"}},
-		"creates only users dir if there are no entries":           {entries: []string{"no entries"}},
+		"user, successfully apply policy trimming whitespaces":           {entries: []string{"entry with spaces"}},
+		"user, successfully apply policy trimming sequential linebreaks": {entries: []string{"entry with multiple linebreaks"}},
+		"user, creates only users_user dir if the entry is empty":        {entries: []string{"entry with no value"}},
+		"user, creates only users dir if there are no entries":           {entries: []string{"no entries"}},
```

Comment:
I think some of them won't depend on the user, like no entries, badly formatted entries?

Reply:
Well, the policy is applied differently if we are doing it for users or for machine

Reply:
you’re correct, this is one of the parameter.